config/environments/test.rb

```diff
@@ -50,6 +50,14 @@
   config.assets.compile = true
   config.assets.digest = false
 
+  config.active_record.verbose_query_logs = true
+
+  config.after_initialize do
+    ActiveRecord::LogSubscriber.backtrace_cleaner.add_silencer do |line|
+      line =~ %r{lib/freedom_patches}
+    end
+  end
```

Comment:
Turn on verbose query logs so that we know which line the query was triggered from.

By default logging is not turned on so this has little to no effect on existing runs.